src/features/swap/index.tsx

```diff
@@ -194,7 +194,7 @@ const SwapWidget = ({ sell }: Params) => {
     if (iframeElement) {
       iframeRef.current = iframeElement as HTMLIFrameElement
     }
-  }, [params])
+  }, [params, isConsentAccepted])
```

Comment:
This makes it work for me when there is a disclaimer but not when reloading the swap page. Adding `safeLoading` here as well seems to make it work though.